clar2wasm/src/tools.rs

```diff
@@ -390,26 +432,35 @@ impl CrossEvalResult {
     }
 }
 
-fn crosseval(snippet: &str, env: TestEnvironment) -> CrossEvalResult {
+fn crosseval(snippet: &str, env: TestEnvironment) -> Result<CrossEvalResult, KnownBug> {
     let mut env_interpreted = env.clone();
     let interpreted = env_interpreted.interpret(snippet);
 
     let mut env_compiled = env;
     let compiled = env_compiled.evaluate(snippet);
 
-    CrossEvalResult {
-        env_interpreted,
-        env_compiled,
-        interpreted,
-        compiled,
+    match KnownBug::check_for_knonw_bugs(&compiled, &interpreted) {
+        Some(bug) => Err(bug),
+        None => Ok(CrossEvalResult {
+            env_interpreted,
+            env_compiled,
+            interpreted,
+            compiled,
+        }),
     }
 }
```

Comment:
nit: move the print in here to avoid dups?

Reply:
Changed in ecd6dbe

Reply:
Damn, I guess my move of the `println!`s was removed after ther rebase :(